doc/tutorials/content/pcl_vcpkg_windows.rst

```diff
@@ -0,0 +1,145 @@
+.. _pcl_vcpkg_windows:
+
+Using PCL on windows with VCPKG and CMake
+-----------------------------------------
+
+This tutorial explains how to acquire Point Cloud Library on
+Microsoft Windows platforms using `VCPKG <https://github.com/microsoft/vcpkg>`_.
+
+For additional information how to use VCPKG, please see their `documentation <https://github.com/microsoft/vcpkg/blob/master/docs/README.md>`_.
+
+Last updated: 22. December 2021
+
+.. image:: images/windows_logo.png
+   :alt: Microsoft Windows logo
+   :align: right
+
+.. contents::
+
+
+Requirements
+==================
+
+Download VCPKG sources to eg. *c:\\vcpkg* preferably by cloning their repository.
+
+Navigate to *c:\\vcpkg* in **powershell** and run 
+
+  .\\bootstrap-vcpkg.bat
+  
+This will download vcpkg.exe.
+
+
+PCL's dependencies
+==================
+
+PCL's required dependencies available on VCPKG are:
+
+* Boost
+* FLANN
+* Eigen3
+
+PCL's optional dependencies available on VCPKG are:
+
+* VTK - for visualization module
+  
+  * Feature OpenGL - required
+  * Feature Qt - optional for QVTK, used in apps
+  
+* GLEW - for simulation module
+* Qhull - for convex/concave in surface module
+* Qt - for apps that use Qt for GUI
+* Google Test - for unit tests
+* Google Benchmark - for benchmarks
+* OpenNI2
+* Realsense2
+* PNG - for a single openni app
+* Pcap - for Velodyne HDL driver
+
+PCL's optional dependencies not on VCPKG
+
+* CUDA - only a port that verify its installed (version 10.1).
+* GLUT
+* OpenNI
+* Ensenso
+* davidSDK
+* DSSDK
+* RSSDK
+
+
+Install PCL for usage
+=====================
+
+Running the following command in powershell in the VCPKG directory,
+will install PCL with default options as well as default triplet type (ie. x86).
+
+  ./vcpkg install pcl
+  
+
+.. note::
+
+  If there are new features or bugfixes that are not yet part of a release,
+  you can try to use --head, which downloads the master of PCL.
+  
+  - 2021-06-25 PCL 11(.1) doesn't support VTK 9, so if you need visualization module, use --head
```

Comment:
```suggestion
```
@larshg Let's remove this and finally merge this pull request :smile:

Reply:
Removed